client/state/stats/lists/test/selectors.js

```diff
@@ -23,10 +23,10 @@ import { userState } from 'state/selectors/test/fixtures/user-state';
 
 describe( 'selectors', () => {
 	beforeEach( () => {
-		getSiteStatsPostStreakData.memoizedSelector.cache.clear();
-		getSiteStatsMaxPostsByDay.memoizedSelector.cache.clear();
-		getSiteStatsTotalPostsForStreakQuery.memoizedSelector.cache.clear();
-		getSiteStatsNormalizedData.memoizedSelector.cache.clear();
+		getSiteStatsPostStreakData.clearCache();
```

Comment:
I think you could use [jest.resetModules()](https://facebook.github.io/jest/docs/en/jest-object.html#jestresetmodules) here instead of `clearCache`.

Reply:
This is an awesome idea, but I'm afraid it requires nontrivial effort to make it work. Instead of importing all the tested selectors once at the top of the test file, we'd have to `require` them in each `beforeEach` call. Otherwise, the `jest.resetModules()` call doesn't have the desired effect.

It's definitely worth exploring in a standalone PR -- there is a lot of `clearCache`-style calls in our tests and they could be all avoided.